packages/vuetify/src/composables/items.ts

```diff
@@ -0,0 +1,56 @@
+// Utilities
+import { computed } from 'vue'
+import { getPropertyFromItem, propsFactory } from '@/util'
+
+// Types
+export interface ItemProps {
+  [key: string]: any
+}
+
+// Composables
+export const makeItemsProps = propsFactory({
+  items: {
+    type: Array,
+    default: () => ([]),
+  },
+  itemText: {
+    type: String,
+    default: 'title',
```

Comment:
Why not align name and default value by changing this to `itemTitle`, the way the others are?